test/grpc/route_guide/route-guide-eventuals-test.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#include <gtest/gtest.h>
+
+#include <string>
+#include <vector>
+
+#include "eventuals/do-all.h"
+#include "eventuals/foreach.h"
+#include "eventuals/grpc/client.h"
+#include "eventuals/range.h"
+#include "test/grpc/route_guide/route-guide-eventuals-server.h"
+#include "test/grpc/route_guide/route_guide.eventuals.h"
+
+using eventuals::grpc::Client;
+using eventuals::grpc::ClientCompletionThreadPool;
+using routeguide::Feature;
+using routeguide::Point;
+using routeguide::Rectangle;
+using stout::Borrowable;
+
+using eventuals::grpc::Stream;
+using grpc::Status;
+
+using eventuals::DoAll;
+using eventuals::Finally;
+using eventuals::Foreach;
+using eventuals::Head;
+using eventuals::Let;
+using eventuals::Range;
+
+////////////////////////////////////////////////////////////////////////
+
+class RouteGuideTest : public ::testing::Test {
+ public:
+  RouteGuideTest() = default;
+
+  int GetPointsCount();
+
+  std::vector<Feature> feature_list_;
+
+  const size_t left_ = -750000000;
+  const size_t right_ = -730000000;
+  const size_t top_ = 420000000;
+  const size_t bottom_ = 400000000;
+  size_t current_ = 0;
+  std::vector<Feature> filtered_;
+  const float kCoordFactor_ = 10000000.0;
+  const int kPoints_ = 10;
+
+  const std::vector<std::tuple<std::string, long, long>> data_ = {
+      {"First message", 0, 0},
+      {"Second message", 0, 1},
+      {"Third message", 1, 0},
+      {"Fourth message", 0, 0}};
+
+ protected:
+  void SetUp() override;
+  void SetDb(const std::string& db);
+
+ private:
+  const std::string db_path_ = "test/grpc/route_guide/route_guide_db.json";
+  const std::string server_address_ = "0.0.0.0";
+  int port_ = 0;
```

Comment:
This is not used except for in a single function, don't make it a "global" here!